cmd/interfaces.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 type runner interface {
-	Dir(d string)
+	SetDir(d string)
```

Comment:
This change is not related to the scope of the pull request which is "Move mock capabilities ..."